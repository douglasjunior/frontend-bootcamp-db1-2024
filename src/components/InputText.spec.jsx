import { describe, expect, it } from '@jest/globals';
import { render } from '@testing-library/react';

import InputText from './InputText';

describe('InputText', () => {
  it('should render the snapshot with default props', () => {
    const result = render(
      <InputText />,
    );

    expect(result).toMatchSnapshot();
  });
});
