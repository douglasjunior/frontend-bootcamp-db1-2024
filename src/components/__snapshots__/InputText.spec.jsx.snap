// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InputText should render the snapshot with default props 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="ant-form-item css-dev-only-do-not-override-14wwjjs ant-form-item-has-success"
      >
        <div
          class="ant-row ant-form-item-row css-dev-only-do-not-override-14wwjjs"
        >
          <div
            class="ant-col ant-form-item-control css-dev-only-do-not-override-14wwjjs"
          >
            <div
              class="ant-form-item-control-input"
            >
              <div
                class="ant-form-item-control-input-content"
              >
                <span
                  class="ant-input-affix-wrapper ant-input-affix-wrapper-status-success css-dev-only-do-not-override-14wwjjs"
                >
                  <input
                    class="ant-input css-dev-only-do-not-override-14wwjjs"
                    type="text"
                    value=""
                  />
                  <span
                    class="ant-input-suffix"
                  >
                    <span />
                  </span>
                </span>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="ant-form-item css-dev-only-do-not-override-14wwjjs ant-form-item-has-success"
    >
      <div
        class="ant-row ant-form-item-row css-dev-only-do-not-override-14wwjjs"
      >
        <div
          class="ant-col ant-form-item-control css-dev-only-do-not-override-14wwjjs"
        >
          <div
            class="ant-form-item-control-input"
          >
            <div
              class="ant-form-item-control-input-content"
            >
              <span
                class="ant-input-affix-wrapper ant-input-affix-wrapper-status-success css-dev-only-do-not-override-14wwjjs"
              >
                <input
                  class="ant-input css-dev-only-do-not-override-14wwjjs"
                  type="text"
                  value=""
                />
                <span
                  class="ant-input-suffix"
                >
                  <span />
                </span>
              </span>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
